test/test-pages/common/sdk-controller.js

```diff
@@ -44,6 +44,17 @@ async function initWebrtcSDK (environmentData, _conversationsApi, noAuth, withDe
     options.defaults.audioStream = audioStream;
     window._defaultAudioStream = audioStream;
 
+    // const formatter = (level, message, details, opts, next) => {
+    //   if (message.includes('propose')) {
+    //     message = 'decorated propose!!!'
+    //   } else {
+    //     message = `[decorated other] ${message}`
+    //   }
+  
+    //   next(level, message, details, opts);
+    // };
+    // options.logFormatters = [formatter];
+
```

Comment:
is this still needed?

Reply:
yes, it's an "uncomment if needed" type of thing.

Reply:
gotcha!